test/run-drun/migrate-stabilization/old-stabilization-source.txt

```diff
@@ -0,0 +1 @@
+old-destabilization.wasm is compiled from `new-destabilzation.mo` with old `moc` compiler using `--sanity-checks`.
```

Comment:
I guess in the integrated branch you could just re-compile in classic mode.

Reply:
Yes, I will prepare this in the integrated PR.